CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+# Contributing to Azure Functions
+
+Azure Functions welcomes contributions and suggestions. One of the easiest ways to contribute is to participate in discussions on GitHub issues. You can also contribute by submitting pull requests with code changes or documentation updates. If you have any questions, please ask a project maintainer.
```

Comment:
What does "please ask a project maintainer" mean? I'm not sure customers will know what this means or how to ask. Could we put something more concrete here?

Reply:
Sure - I guess I'm not sure what the right call is there. Discussion in issues or on PRs seems like the most direct way. Thoughts?

Reply:
Would it be worthwhile to create a GitHub issue template for asking questions, and directing people there? Some repos also use [Gitter](https://gitter.im/) for discussion, though I'm not sure if that's another forum that we want to have to start dealing with.